closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -678,7 +931,7 @@ JNIEXPORT jlong JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_createRSAPublic
         return -1;
     }
 
-    RSA* publicRSAKey = RSA_new();
+    RSA* publicRSAKey = (*OSSL_RSA_new)();
```

Comment:
This won't compile: declarations must be at the beginning of a block.

Reply:
This does compile !!  
Leaving as-is as this was Nasser's code and I don't want to change it when it works.
I also like defining things as close to where they are first used as it makes the code more readable.

Reply:
This might not compile on jdk8 on windows.

Reply:
Understand, but this code is for JDK11

Reply:
It isn't about jdk8 or jdk11: it isn't strictly legal in C to declare variables anywhere but at the beginning of a block and some compilers (notably MSVC) won't accept such illegal code. The code in jdk8 was fixed - this should be fixed similarly.